src/main/features/response/views/statement-of-means/debts.njk

```diff
@@ -0,0 +1,118 @@
+</span>
+
+{% endmacro %}
+
+{% macro debtRow(index, form) %}
+
+  {% set prefix = 'rows[' + index + ']' %}
+
+  {% set debtKey = prefix + '[debt]' %}
+  {% set debtValue = form.valueFor(debtKey) %}
+  {% set debtError = form.errorFor(debtKey) %}
+
+  {% set totalOwedKey = prefix + '[totalOwed]' %}
+  {% set totalOwedValue = form.valueFor(totalOwedKey) %}
+  {% set totalOwedError = form.errorFor(totalOwedKey) %}
+
+  {% set monthlyPaymentsKey = prefix + '[monthlyPayments]' %}
+  {% set monthlyPaymentsValue = form.valueFor(monthlyPaymentsKey) %}
+  {% set monthlyPaymentsError = form.errorFor(monthlyPaymentsKey) %}
+
+  {% set anyErrorOccurred = debtError or totalOwedError or monthlyPaymentsError %}
+  <div class="triple-column{% if anyErrorOccurred %} form-group-error {% endif %}">
+
+    {% if debtError %}<span id="{{ debtKey }}[label]" class="error-message">{{ t(debtError) }}</span>{% endif %}
+    {% if totalOwedError %}<span id="{{ totalOwedKey }}[label]" class="error-message">{{ t(totalOwedError) }}</span>{% endif %}
+    {% if monthlyPaymentsError %}<span id="{{ monthlyPaymentsKey }}[label]" class="error-message">{{ t(monthlyPaymentsError) }}</span>{% endif %}
+
+    <label class="form-label-bold mobile-show" for="{{ debtKey }}">{{ index + 1 }} {{ t('. List each debt') }}</label>
+    {{ textInputInline(name = debtKey, form = form, inputClass='column-1') }}
+    {{ multiRowPoundInput(name = totalOwedKey, label = (index + 1) + '. Total owed', form = form,  cssClass = 'column-4') }}
+    {{ multiRowPoundInput(name = monthlyPaymentsKey, label = (index + 1) + '. Monthly payments', form = form, cssClass = 'column-3') }}
+  </div>
+
+{% endmacro %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+
+    <p>{{ t('For example credit cards, loans or if you’re behind on rent or utility bills.') }}</p>
+
+    <form method="post" novalidate
+          class="analytics-submit-event-trigger"
+          data-event-action="SoM: Do you have any debts?"
+          data-event-label-from="hasAnyDebts">
+
+      {{ csrfProtection(csrf) }}
+
+      {% set errorHasAnyDebts = form.errorFor('hasAnyDebts') %}
+      <div class="statement-of-means-container">
+        <fieldset id="hasAnyDebts[label]" class="form-group {% if errorHasAnyDebts %} form-group-error {% endif %}">
+          <legend class="visually-hidden">{{ t('Do you have any debts?') }}</legend>
+
+          {{
+            radioOption(form = form, label = 'Yes', name = 'hasAnyDebts', value = 'true', checked = form.valueFor('hasAnyDebts') === true)
+          }}
+
+          {% set noRowsError = form.errorFor('rows') %}
+          <div class="form-group panel panel-border-narrow js-hidden {% if noRowsError %} form-group-error {% endif %}"
+               id="hasAnyDebts-true" aria-hidden="false">
+
+            {% if noRowsError %}<span class="error-message" id="rows[label]">{{ t(noRowsError) }}</span>{% endif %}
+
+            <div class="timeline-rows">
```

Comment:
timeline-rows?

Reply:
the thing that we reuse everywhere :/ Should be renamed to more generic thing. 

I hope I will have time to retrospectively refactor old multi-row forms (such us claim amount, evidence and timeline). I want to change this css then as well.